Quadriga/src/main/java/edu/asu/spring/quadriga/validator/WorkspaceFormValidator.java

```diff
@@ -11,42 +11,36 @@
 
 /**
  * This class validates if any workspace is selected
+ * 
  * @author kiran batna
  *
  */
 @Service
 public class WorkspaceFormValidator implements Validator {
 
-	@Override
-	public boolean supports(Class<?> arg0) {
-		return arg0.isAssignableFrom(ModifyWorkspaceForm.class);
-	}
-
-	@Override
-	public void validate(Object obj, Errors err) 
-	{
-		List<ModifyWorkspace> workspaceList;
-		String internalId;
-		boolean isAllNull = true;
-		
-		ModifyWorkspaceForm workspaceForm = (ModifyWorkspaceForm)obj;
-		workspaceList = workspaceForm.getWorkspaceList();
-		
-		//validating if any row is selected
-		for(ModifyWorkspace workspace : workspaceList)
-		{
-			internalId = workspace.getId();
-			if(internalId != null)
-			{
-				isAllNull = false;
-			}
-		}
-		//if no row is selected
-		if(isAllNull)
-		{
-			err.reject("workspace_selection.required");
-		}
-
-	}
+    @Override
+    public boolean supports(Class<?> arg0) {
+        return arg0.isAssignableFrom(ModifyWorkspaceForm.class);
+    }
+
+    @Override
+    public void validate(Object obj, Errors err) {
+        boolean isAllNull = true;
+
+        ModifyWorkspaceForm workspaceForm = (ModifyWorkspaceForm) obj;
+        List<ModifyWorkspace> workspaceList = workspaceForm.getWorkspaceList();
+
+        if (workspaceList != null) {
+            // validating if any row is selected
+            for (ModifyWorkspace workspace : workspaceList) {
+                String internalId = workspace.getId();
+                if (internalId != null) {
+                    isAllNull = false;
+                }
+            }
+            err.reject("workspace_selection.required");

```

Comment:
@HariChandanaKanchanapally Can you please delete the file after completing activate workspace